### PR TITLE
Rename Timestream Aggregator to Smurf Recorder

### LINKED ISSUE #49: rename the timestream aggregator
@mhasself points out that "timestream-aggregator" is perhaps too general/not quite accurate. We should think of a better name for this agent. Please brainstorm ideas here.

### PR DESCRIPTION
**Pull request by BrianJKoopman**
resolves #49

### CODE REVIEW

**Comment by mhasself** [comment]
What about the docs?

**Comment by BrianJKoopman** [comment]
Right, maybe too aggressively `grep -v`'d. New commit pushed, also removed other aggregator terminology related to the names of tasks/processes.

File: agents/smurf_recorder/smurf_recorder.py [hunk 1]
```diff
@@ -154,8 +154,8 @@ def make_parser(parser=None):
                              port=int(args.port))
 
     agent.register_process("stream",
```
[the remark this answers is not in the document]
Comment (BrianJKoopman) [reply agents/smurf_recorder/smurf_recorder.py]: Good point, changed to "record".

File: agents/smurf_recorder/smurf_recorder.py [hunk 2]
```diff
@@ -70,12 +70,12 @@ def __init__(self, agent, time_per_file, data_dir, address="localhost",
...
 
-    def start_aggregation(self, session, params=None):
-        """start_aggregation(params=None)
+    def start_recording(self, session, params=None):
+        """start_recording(params=None)
 
-        OCS Process to start data aggregation. This Process uses FrameRecorder,
-        which deals with I/O, requiring this process to run in a worker thread.
-        Be sure to register with blocking=True.
+        OCS Process to start recording SMuRF data. This Process uses
```
[the remark this answers is not in the document]
Comment (BrianJKoopman) [reply agents/smurf_recorder/smurf_recorder.py]: I went with "start_record", as other agents' methods are typically a "start_X" process and "stop_X" task.

**Pull request merged**